packages/front-end/components/Experiment/ImportExperimentList.tsx

```diff
@@ -186,7 +197,11 @@ const ImportExperimentList: FC<{
                           )?.name
                         : "experiments"}
                     </td>
-                    <td>{e.trackingKey}</td>
+                    {isExperimentName ? (
+                      <td>{e?.experimentName}</td>
+                    ) : (
+                      <td>{e.trackingKey}</td>
+                    )}
```

Comment:
This could break the existing page.  Just because they set the `hasNameCol` toggle doesn't mean the past experiment results in Mongo are immediately up-to-date with the newest columns.  They would still have to go in and refresh that list to get the new name columns.

In fact, we don't really care about the `hasNameCol` setting at all.  No matter what, new past experiment rows will always have an `experimentName` field defined.  And for old rows already stored in Mongo before this PR, we can fall back to using `trackingKey`.

So we can just do this instead:

```tsx
<td>{e.experimentName || e.trackingKey}</td>
```

Reply:
Also, further down this file on line ~222, we should use the experiment and variation names.

So for example
```tsx
const importObj: Partial<ExperimentInterfaceStringDates> = {
  name: e.trackingKey,
```
should become
```tsx
const importObj: Partial<ExperimentInterfaceStringDates> = {
  name: e.experimentName || e.trackingKey,
```

Reply:
Is the current use good?